brainstormforce/astra-import-export
# AST-1889 - PHP notice while importing the settings - Undefined index astra-addons

### Description
- PHP Notice displayed when importing the customizer settings.


### Screenshots


### Types of changes

 Bug fix (non-breaking change which fixes an issue) 



### How has this been tested?
Steps to Reproduce:
- Install fresh Astra theme no Astra Addon needed.
- Export the settings
- Add the logs in wp-config.php

define( 'WP_DEBUG', true ); // To make dev environment.
define( 'WP_DEBUG_LOG', true ); // To make debug.log file in wp-content.

- Now import the settings and check the debug log file.


[25-May-2022 07:26:41 UTC] PHP Notice:  Undefined index: astra-addons in C:\Users\niles\projects\wp60\app\public\wp-content\plugins\astra-import-export\inc\classes\class-astra-import-export-loader.php on line 167
[25-May-2022 07:26:41 UTC] PHP Stack trace:
[25-May-2022 07:26:41 UTC] PHP   1. {main}() C:\Users\niles\projects\wp60\app\public\wp-admin\themes.php:0
[25-May-2022 07:26:41 UTC] PHP   2. require_once() C:\Users\niles\projects\wp60\app\public\wp-admin\themes.php:10
[25-May-2022 07:26:41 UTC] PHP   3. do_action() C:\Users\niles\projects\wp60\app\public\wp-admin\admin.php:175
[25-May-2022 07:26:41 UTC] PHP   4. WP_Hook->do_action() C:\Users\niles\projects\wp60\app\public\wp-includes\plugin.php:476
[25-May-2022 07:26:41 UTC] PHP   5. WP_Hook->apply_filters() C:\Users\niles\projects\wp60\app\public\wp-includes\class-wp-hook.php:331
[25-May-2022 07:26:41 UTC] PHP   6. Astra_Import_Export_Loader->import() C:\Users\niles\projects\wp60\app\public\wp-includes\class-wp-hook.php:307


### Checklist:
